assets/scss/_404.scss

```diff
@@ -1,16 +1,16 @@
 .p404-page {
   .p404 {
-		&__content-wrapper {
-			height: 350px;
-			@media (max-width: 991px) {
-	      height: 600px;
+    &__content-wrapper {
+      height: 350px;
```

Comment:
still broken indentations. Rules need to be added

Reply:
Think I'm missing something, it looks like indentations were fixed after I ran "npm run fix-style"